code/lib/theming/src/utils.ts

```diff
@@ -63,9 +60,9 @@ export const darkenColor = colorFactory('darken');
 // The default color scheme is light so unless the preferred color
 // scheme is set to dark we always want to use the light theme
 export const getPreferredColorScheme = () => {
-  if (!globalWindow || !globalWindow.matchMedia) return 'light';
+  if (!globalThis || !globalThis.matchMedia) return 'light';
```

Comment:
I believe `globalThis` is supported in all node and browser versions that we are currently targeting, and it makes mocking in the tests much easier.

Reply:
It would be interesting to try and replace `@storybook/global` with `globalThis` everywhere someday.